scripts/scripts.js

```diff
@@ -0,0 +1,10 @@
+//Create click handlers that alerts the dog’s information when clicking on the dog’s photo.
+//You will need to pass in the params in the click handler of the dog’s name, breed and adoption fees.
+//Create click handlers on each dog’s button that will add to the total.
+//Alert this total.
+
+let x = myFunction();
+function myFunction () {
+  alert(//something);
```

Comment:
Not relevant for this assignment but you are getting a js error because you commented out  the entire line. 
This also commented out the parenthesis and every bracket type in code must be closed properly.